core/jazz_ui/src/config/templates/java-yaml.ts

```diff
@@ -1,8 +1,11 @@
 export const javaTemplate = {
-    template:
-`functions:
-    function1:
-      handler: function1/com.slf.services.Handler
-    function2:
-      handler: function2/com.slf.services.Handler`
+  template:
+`package:
+artifact: target/sls-app-java-template-1.0.jar
```

Comment:
Fix indentation

Reply:
make sure u r using spaces and not tabs. also there is a ` character before package which doesn't belong there.

Reply:
In the yaml, make sure not to use tabs, just use spaces.

Reply:
fixed this